canonical-web-and-design/maas-ui
# feat(ui): add generic networking fields

## Done

- Add a number of fields for networking forms.
- Update the add machine form to use the mac address
field.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list and open the add machine form.
- Check that the mac address fields works as before.

## Fixes

Fixes: #2185.
